traits/src/types.rs

```diff
@@ -181,7 +181,7 @@ pub enum HpkeKemType {
     DhKem448 = 0x0021,
 
     /// XWing combiner for ML-KEM and X25519
-    XWingKemDraft2 = 0x004D,
+    XWingKemDraft6 = 0x004D,
```

Comment:
This is a breaking change. We need to bump the version for this crate and add it to the changelog.